INN/umbrella-ipbs
# Add grid template columns repeat to our impact section

## Changes

This pull request makes the following changes:

- Adds `grid-template-columns: repeat(3, 1fr);` attribute to the our impact grid widget area to make all boxes the same width

Before:
![Screen Shot 2020-06-25 at 11 52 35 AM](https://user-images.githubusercontent.com/18353636/85753680-74631e00-b6da-11ea-9680-46ed3b007bc2.png)

After:
![Screen Shot 2020-06-25 at 11 52 17 AM](https://user-images.githubusercontent.com/18353636/85753683-74fbb480-b6da-11ea-9069-156dffe80845.png)

## Why



For #93

## Testing/Questions

Features that this PR affects:

- Homepage "our impact" section


Questions that need to be answered before merging:


Steps to test this PR:

1. View the boxes (make sure they have images with the same aspect ratio) and make sure they all are the same width
